module6/freethrows/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Free Throws
+---
+
+During Intermission, you worked on earning a certification in fundamental C#.  This is one way to keep up with your technical skills during a job hunt.  Another way to stay current with your skills is by practicing 'Free Throws' - activities that will help you cement your existing skills into muscle memory.
+
+During Mod6 we will have some dedicated Free Throw time that will focus on one of three areas: programming fundamentals, OOP Practice, and SQL Practice.  
+
+<section class='note' markdown='1'>
+Keep revisiting this page - we may add more opportunities for practice as we go!
+</section>
+
+### Programming Fundamentals
```

Comment:
Random idea, because this time is so short, my initial thought is we want it to feel sort of like a sprint, like a take-home coding challenge sort of energy. What do you think about at the start of free throws posting a thread where students set a goal for themselves for this morning's free throw time, then at the end another thread saying if they met their goal. And of course reiterating that you won't always meet your goal, totally ok. But good to have something you're shooting for.

Also up for not having more structure, but I think some focus would be helpful here because the time is so short and it's every day.

Reply:
I like this structure!  Especially for the first couple of weeks :)